packages/calcite-components/src/tests/commonTests.ts

```diff
@@ -975,6 +975,14 @@ interface DisabledOptions {
    *  Note: this should only be used for components that wrap a single component that implements disabled behavior.
    */
   shadowAriaAttributeTargetSelector?: string;
+
+  /**
+   * Use this to specify if any pointerEvent is disabled on click.
+   *
+   *  Note: mousedown and mouseup events are skipped when pointerEvent has preventDefault
+   *  https://github.com/web-platform-tests/wpt/blob/master/pointerevents/compat/pointerevent_mouse-pointer-preventdefault.html#L54
+   */
+  pointerEventPreventDefaultOnClick?: boolean;
```

Comment:
This argument is for a very particular use case and I'm not sure about its wider applicability.

Reply:
Yup, it is a specific use case . Added this to the options so that we can extend this if required to any component in future.  `pointerDownEventDefaultPrevented` might be a better name?